lib/sync/datasetClientsCleaner.js

```diff
@@ -22,26 +23,42 @@ var storage;
 function DatasetClientsCleaner(opts) {
   this.retentionPeriod = opts.retentionPeriod || '24h';
   this.checkFrequency = opts.checkFrequency || '1h';
+  this.cleanerLockName = opts.cleanerLockName || 'locks:sync:DatasetCleaner';
```

Comment:
it is, but TBH I don't think user will ever need to change it. Is there really value there to allow user to config it?